indymeet/templates/registration/login.html

```diff
@@ -23,13 +23,7 @@
                 </div>
             </div>
         {% endif %}
-        {% if form.errors and not form.non_field_errors %}
-          <div class="alert alert-danger">
-          {% for key, error in form.errors.items %}
-          <p> {{ key }}:{{ error }}</p>
-          {% endfor %}
-          </div>
-          {% endif %}
+        {% include 'includes/no_field_errors.html' %}
```

Comment:
Wait, isn't this only rendering the `non_field_errors` now though?

Reply:
Yes but the other errors are rendered under the fields, I don't think we need to put them twice.

Reply:
Ah, that makes sense. Good catch